src/site/index.ts

```diff
@@ -1,7 +1,9 @@
 // @ts-expect-error @moduk/frontend does not yet support types
 import * as mod from '@moduk/frontend/client'
+import { initCopyButtons } from './_includes/components/code-snippet'
```

Comment:
To discuss: I thought initCopyButtons was a meaningful name here. Also, I know sometimes linting rules say to use it, but I don't like default exports so much as they can lead to inconsistent names and you have to change it if you add another export.

Reply:
I agree, `navigation()` and `copyButtons()` don't really mean anything. I will update in #23

Reply:
Cool, I will probably rename this one to `initCodeSnippets` also for consistency.